### ISSUE TITLE
[Adaptive Card iOS - Image.BackgroundColor.json]: Voice over does not announce the descriptive information about the image.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8406]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.1" control and select it.
5. Navigate to 'Test' control and activate it.
6. Navigate to 'Image.BackgroundColor.json' and activate it.
7. Navigate to the image and observe the issue.

**Actual Result:**
While navigating to the image under 'Image.BackgroundColor.json' screen, Screen Reader does not convey the descriptive information about image. Currently it is announcing as 'Image' only.

**Expected Result:**
Voice over should convey the descriptive information to the user, and it should announce as **'cat image with red background color'**.

**Note:**
This issue is repro throughout the application for the same type of controls.

**User Experience:**
It will impact the screen reader user if the screen reader does not announce the descriptive information to the user about the image as user will get confused if the same name as image is announced for all images.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/non-text-content

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/227844221-3f81cbfd-8c64-42da-b563-b667a6766dd0.MP4

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
closing this issue, the app wasn't built correctly perhaps, only cards under Scenario should be tested, not under Elements. I will make note of this and fix it.

**Comment by vagpt** [comment]
Adding A11yTTValidated tag as per the above comment.
